NineChronicles.Headless/GraphTypes/RpcInformationQuery.cs

```diff
@@ -19,6 +20,34 @@ public RpcInformationQuery(ActionEvaluationPublisher publisher)
                 name: "clients",
                 description: "List of address connected to this node.",
                 resolve: context => publisher.GetClients());
+            Field<NonNullGraphType<IntGraphType>>(
+                name: "totalCountByDevice",
+                arguments: new QueryArguments(
+                    new QueryArgument<NonNullGraphType<StringGraphType>>
+                    {
+                        Name = "device"
+                    }
+                ),
+                description: "total count by connected to this node.",
+                resolve: context =>
+                {
+                    string device = context.GetArgument<string>("device");
+                    return publisher.GetClientsCountByDevice(device);
+                });
+            Field<NonNullGraphType<ListGraphType<AddressType>>>(
```

Comment:
If `GetClientsByDevice` cannot a list including `null`, it can guarantee it in type-level.

```suggestion
            Field<NonNullGraphType<ListGraphType<NonNullGraphType<AddressType>>>>(
```